electron-app/src/views/settings/Accounting.vue

```diff
@@ -140,6 +149,12 @@ export default class Accounting extends Vue {
   assetToIgnore: string = '';
   assetToRemove: string = '';
 
+  taxFreeRules = [
+    (v: string) => !!v || 'Please enter the number of days',
+    (v: string) =>
+      (v && parseInt(v) > 0) || 'The number of days cannot negative or zero'
```

Comment:
Very nice ...thanks to this I realized I was not checking for 0 in the API and a 0 value (which is bad) could end up in the DB: https://github.com/rotki/rotki/pull/688

-1 is a valid value for the API, to disable it but from the front-end this is already done when flipping the switch. Should not be a valid value apart from disabling. So the check is good.

Perhaps just add the missing "be" there, or I will do it.

Reply:
Btw there was already a warning about having a zero in db.

Reply:
yeah I guess this is how you saw it. Now it should not be possible in the API level too.